src/_premake_init.lua

```diff
@@ -467,6 +467,18 @@
 	}
 
 	api.register {
+		name = "inlining",
+		scope = "config",
+		kind = "string",
+		allowed = {
+			"Default",
+			"Disabled",
+			"Explicit",
+			"Implicit"
```

Comment:
I don't think it's super-obvious what 'Implicit' mean in this context.
The other names do make sense to me, but I'm not sure they're the best possible names either.